src/libdeno.rs

```diff
@@ -19,20 +19,20 @@ pub struct deno_buf {
   pub data_len: usize,
 }
 
-type DenoRecvCb = unsafe extern "C" fn(d: *const DenoC, buf: deno_buf);
+type DenoRecvCb = unsafe extern "C" fn(d: *const isolate, buf: deno_buf);
 
 extern "C" {
   pub fn deno_init();
   pub fn deno_v8_version() -> *const c_char;
-  pub fn deno_set_flags(argc: *mut c_int, argv: *mut *mut c_char);
-  pub fn deno_new(data: *const c_void, cb: DenoRecvCb) -> *const DenoC;
-  pub fn deno_delete(d: *const DenoC);
-  pub fn deno_last_exception(d: *const DenoC) -> *const c_char;
-  pub fn deno_get_data(d: *const DenoC) -> *const c_void;
-  pub fn deno_set_response(d: *const DenoC, buf: deno_buf);
-  pub fn deno_send(d: *const DenoC, buf: deno_buf);
+  pub fn deno_set_v8_flags(argc: *mut c_int, argv: *mut *mut c_char);
+  pub fn deno_new(data: *const c_void, cb: DenoRecvCb) -> *const isolate;
+  pub fn deno_delete(d: *const isolate);
+  pub fn deno_last_exception(d: *const isolate) -> *const c_char;
+  pub fn deno_get_data(d: *const isolate) -> *const c_void;
+  pub fn deno_set_response(d: *const isolate, buf: deno_buf);
+  pub fn deno_send(d: *const isolate, buf: deno_buf);
```

Comment:
Many of these functions use the parameter name `d` for `*const isolate`. It would be better to rename this to either `i` or `isolate_ptr`.